### DISCUSSION TITLE
part-ii/2016-125/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#325] [Question comments]
Part II 2016 Riemann Surfaces: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2016-125/

**Comment by eha1234** [comment]
For the second part, note that $f$ is at most degree 3 (write it out). You can find two roots, namely $v$ and $w$. Now the third root must satisfy the relation, using the first part and considering where the poles are.
